getting-started/bare-metal/about.md

```diff
@@ -1,56 +1,64 @@
+
+- All platforms in this release except Windows
 
-This means that if you are using {{site.prodname}} networking in your cluster, and you want the non-cluster hosts to be able to communicate with workloads in the cluster, you will need to install {{site.prodname}} for networking on your non-cluster hosts.  If you are not using {{site.prodname}} for networking in your cluster (e.g. are using {{site.prodname}} in policy-only mode), or don't need your non-cluster hosts to communicate directly with workloads, you can install {{site.prodname}} in policy-only mode on your non-cluster hosts.
+**Required**
 
-#### {{site.prodname}} policy on non-cluster hosts
+- Review your install options
 
-{{site.prodname}} policy allows you to control firewalls on your non-cluster hosts using the same controls powerful controls as in your cluster.  {{site.prodname}} must be running on each non-cluster host you want to control.
+| {{site.prodname}} options     | Requires                 | Use case                                                     | Supported install methods                      |
+| ----------------------------- | ------------------------ | ------------------------------------------------------------ | ---------------------------------------------- |
+| Networking and network policy | Calico CNI               | **Networking**<br />You want {{site.prodname}} networking to handle pod-to-host, and host-to-pod communications.<br /><br />**Policy**<br />You want to use {{site.prodname}} network policy to control firewalls on your non-cluster hosts. | Docker container                               |
+| Policy only                   | A supported provider CNI | You are using a supported provider CNI for networking (for example, EKS with AWS CNI), but want to use {{site.prodname}} network policy. | Binary install with/ without a package manager |
+|                               | Calico CNI               | You are using the Calico CNI, but your non-cluster hosts do not need to communicate with pods. |                                                |  
 
-Using {{site.prodname}}, you can secure network interfaces of the host; these interfaces are called **host endpoints** (to distinguish them from **workload endpoints**). Host endpoints can have labels, which work the same as labels on workload endpoints. This allows you to create {{site.prodname}} network policy for either host or workload endpoints, where each selector can refer to the either type (or be a mix of the two) using labels.
+- Host meets [system requirements](./requirements) for {{site.prodname}}. If you want to use a package manager for installation, the host must be an Ubuntu- or RedHat-derived system.
+- Set up a datastore; if {{site.prodname}} is installed on a cluster, you already have a datastore
+- `kubectl` is installed on the cluster if you are using the Kubernetes datastore (default), or [`calicoctl`]({{site.baseurl}}/getting-started/clis/calicoctl/) for an etcd3 datastore
```

Comment:
calicoctl still works for kubernetes datastore, fwiw.

Reply:
Adjusted text.